File: itsm/src/main/kotlin/co/brainz/workflow/process/ProcessHistRepository.java
package co.brainz.workflow.process;

import java.util.List;

import org.springframework.data.jpa.repository.JpaRepository;

public interface ProcessHistRepository extends JpaRepository<ProcessHist, String> {
    List<ProcessHist> findByProcId(String procId);
    List<ProcessHist> findAll();

}

File: itsm/src/main/kotlin/co/brainz/workflow/process/ProcessHistRepository.kt
package co.brainz.workflow.process

import org.springframework.data.jpa.repository.JpaRepository;

public interface ProcessHistRepository : JpaRepository<ProcessHist, String> {
	fun findByProcId(procId : String) : MutableList<ProcessHist>
	override fun findAll() : MutableList<ProcessHist>
}
